packages/runtime/src/components/builtin/Countdown/Countdown.tsx

```diff
@@ -71,7 +69,6 @@ const Container = styled.div<{
   labelFont?: ResponsiveValue<string>
 }>`
   display: flex;
-  ${cssWidth('560px')}
```

Comment:
And this `560px`?

Reply:
It's inferred automatically from the control's default value 😍